templates/addons/aurora/postgresql-cf.yml

```diff
@@ -0,0 +1,153 @@
+Parameters:
+  App:
+    Type: String
+    Description: Your application's name.
+  Env:
+    Type: String
+    Description: The environment name your service, job, or workflow is being deployed to.
+  Name:
+    Type: String
+    Description: The name of the service, job, or workflow being deployed.
+  # Customize your Aurora Serverless cluster by setting the default value of the following parameters.
+  InitialDBName:
+    Type: String
+    Description: The name of the initial database to be created in the DB cluster.
+    Default: {{.InitialDBName}}
+    # Cannot have special characters
+    # Naming constraints: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Limits.html#RDS_Limits.Constraints
+  DBMinCapacity:
+    Type: Number
+    Description: The minimum scaling capacity for the cluster.
+    AllowedValues: [2, 4, 8, 16, 32, 64, 192, 384]
+    Default: 2
+  DBMaxCapacity:
+    Type: Number
+    Description: The maximum scaling capacity for the cluster.
+    AllowedValues: [ 2, 4, 8, 16, 32, 64, 192, 384 ]
+    Default: 8
+  DBAutoPauseSeconds:
+    Type: Number
+    Description: The duration in seconds before the cluster pauses.
+    Default: 1000
+Resources:
+  AuroraKMSCMK:
```

Comment:
If we are not doing secret rotation for now, do we still need KMS?

Reply:
I don't know if i asked this anywhere - but do we need this given that we have an env kms key?

Reply:
This is used to encrypt the database instances so it has nothing to do with secret rotation I believe.

Reply:
We have an app kms key but nothing at the env level I think. We could use the default key but that might run afoul of compliance issues. But that also seems to imply that folks would roll their own CF template if they needed a dedicated key.

Reply:
Oh I see but can we possibly reuse the KMS we created [here](https://github.com/aws/copilot-cli/blob/1b33c83e488047d5a7a42918bef4775464807519/templates/app/cf.yml#L13-L17)?

Reply:
> We have an app kms key but nothing at the env level I think. We could use the default key but that might run afoul of compliance issues. But that also seems to imply that folks would roll their own CF template if they needed a dedicated key.

That makes sense to me. I feel we should put it in the env stack so that this KMS can be shared. The reason is because for features like `exec`, it allows users to encrypt by KMS (though we didn't do it in our first milestone).

Reply:
Update: app level KMS is used by default. User can customize the KMS field if they have more advanced use case.